x-pack/plugins/enterprise_search/public/applications/shared/licensing/manage_license_button.tsx

```diff
@@ -32,6 +32,7 @@ export const ManageLicenseButton: React.FC<EuiButtonProps> = (props) => {
       target="_blank"
       iconType="popout"
       href={`${docLinks.enterpriseSearchBase}/license-management.html`}
+      data-test-subj="manageLicenseButton"
```

Comment:
I see this was also added as a part of 1d2e2de60b0d71402b438ef379ef7374eda3a016 seems unrelated again

Reply:
https://github.com/elastic/kibana/pull/117242/commits/fd9f20de4a5a20ecafbf30d1146821ced94e7122 - also apparently this specific file has not been open in my VSCode for months now so this is legit baffling haha